docker-compose.yaml

```diff
@@ -18,6 +18,9 @@ services:
     command: bash -c "wait-for-it -s -t 60 postgres:5432 && wait-for-it -s -t 60 redis:6379 -- usedgoodreads"
     ports:
       - "5000:5000"
+    volumes:
+      - /etc/letsencrypt/live/usedgoodreads.com/fullchain.pem:/etc/letsencrypt/live/usedgoodreads.com/fullchain.pem
+      - /etc/letsencrypt/live/usedgoodreads.com/privkey.pem:/etc/letsencrypt/live/usedgoodreads.com/privkey.pem
```

Comment:
I don't understand these lines; this base docker compose will get used for both local development as well as production. For local development, I don't have a letsencrypt setup at `/etc/letsencrypt/..` and not sure we should require this?